content/terraform-docs-common/docs/cloud-docs/api-docs/private-registry/stack-component-configurations.mdx

```diff
@@ -0,0 +1,689 @@
+        }
+      },
+      "resources": {
+        "links": {
+          "related": "/api/v2/registry-components/comp-FRxdjivJqVVfi2SY/versions/1.0.0/resources"
+        }
+      },
+      "variables": {
+        "links": {
+          "related": "/api/v2/registry-components/comp-FRxdjivJqVVfi2SY/versions/1.0.0/variables"
+        }
+      },
+      "outputs": {
+        "links": {
+          "related": "/api/v2/registry-components/comp-FRxdjivJqVVfi2SY/versions/1.0.0/outputs"
+        }
+      },
+      "providers": {
+        "links": {
+          "related": "/api/v2/registry-components/comp-FRxdjivJqVVfi2SY/versions/1.0.0/providers"
+        }
+      },
+      "subcomponents": {
+        "links": {
+          "related": "/api/v2/registry-components/comp-FRxdjivJqVVfi2SY/versions/1.0.0/subcomponents"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/registry-components/compver-Ecdq2uaNUXNHNHcA/versions/1.0.0",
+      "upload": "https://archivist.terraform.io/v1/object/dmF1bHQ6djE6Q1JjVWM5ZHBFRXZzdWZTKy80NUh3bDRMREd4QkxRaXZ0bXE3QXYzRHFMcFE0NVVhVmRmSE43K0dTUlp1aUsxQ253VmN6SFoyM1pLWU4vbGxsVW8wUkh4UzZ3S1ZmYXljRHhKM1gxR0RIdTJiSlNMbGYzYmo5ejVlUHZPQlArRUJHa1h3WUFLemhuOUhsS3pXZnRvbHppbVJYZEJCUjl6TlZLdEhqT29DdjY4U3RQRW51MkptTCsvUUlWK2xKNDhjK2NaT1JCbjNLYWtGZFh1UGtJL2V4YVFYbjVEVXdScEpJNzdrR2ZKSXgvNjd6UkVEZDBDZElBQnljK0Y2YnhVeUpUV1BvRWdkY0hPajhyWHdKNW1QTS93bnNHK0lSSUVoQ05OTGJaR09MMnRzeHl4ZzBST2VVaG9PQzV6YjR5WXprU0REMldPSUxTV0xRVlBRbldvaGtRVUFtUWU3QitrWERoOUUwaFczOG1rTG4xc21KZ240ZU40PQ"
+    }
+  }
+}
+```
+
+## Add a Stack Component Configuration Version
+
+`PUT https://archivist.terraform.io/v1/object/<UNIQUE OBJECT ID>`
+
+**The URL is provided in the `upload` links attribute in the `registry-component-versions` resource.**
+
+### Expected Archive Format
+
+HCP Terraform expects the component configuration version uploaded to be a gzip tarball with the component configuration in the root (not in a subdirectory).
+
+Given the following folder structure:
+
+```
+terraform-component-eks-cluster
+├── README.md
+├── examples
+│   └── default
+│       ├── README.md
+│       └── main.tf
+└── components.tfcomponent.hcl
+```
+
+Package the files in an archive format by running `tar zcvf component.tar.gz *` in the component configuration's directory.
+
+```
+~$ cd terraform-component-eks-cluster
+terraform-component-eks-cluster$ tar zcvf component.tar.gz *
+a README.md
+a examples
+a examples/default
+a examples/default/main.tf
+a examples/default/README.md
+a components.tfcomponent.hcl
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/octet-stream" \
+  --request PUT \
+  --data-binary @component.tar.gz \
+  https://archivist.terraform.io/v1/object/dmF1bHQ6djE6NWJPbHQ4QjV4R1ox...
+```
+
+After the Stack component configuration version is successfully parsed, its status will become `"completed"`.
+
+## Get a Stack Component Configuration
```

Comment:
Does this return multiple versions? It might be helpful to add a sentence to specify